core/src/test/scala/unit/kafka/zk/ZkMigrationClientTest.scala

```diff
@@ -49,12 +51,25 @@ class ZkMigrationClientTest extends QuorumTestHarness {
 
   private var migrationState: ZkMigrationLeadershipState = _
 
+  private val SECRET = "secret"
+
+  private val encoder: PasswordEncoder = {
+    val encoderProps = new Properties()
+    encoderProps.put(KafkaConfig.ZkConnectProp, "localhost:1234") // Get around the config validation
+    encoderProps.put(KafkaConfig.PasswordEncoderSecretProp, SECRET) // Zk secret to encrypt the
+    val encoderConfig = new KafkaConfig(encoderProps)
```

Comment:
I don't think we need these anymore, right?

Reply:
Oh, I see -- we're getting the defaults for things like the cipher, key length, etc. Let's keep this in, in that case